WordPressKit/RemotePost.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import ObjectiveC
+
+@objcMembers public class RemotePost: NSObject {
+
+    public static let statusDraft = "draft"
+    public static let statusPending = "pending"
+    public static let statusPrivate = "private"
+    public static let statusPublish = "publish"
+    public static let statusScheduled = "future"
+    public static let statusTrash = "trash"
+    public static let statusDeleted = "deleted" // Returned by wpcom REST API when a post is permanently deleted.
```

Comment:
What do you think of making that comment a documentation one?

```suggestion
    /// Returned by the WordPress.com REST API when a post is permanently deleted.
    public static let statusDeleted = "deleted"
```